core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/state/ReplicationState.java

```diff
@@ -60,4 +94,36 @@ private LongTokenRange convert(TokenRange range)
         long end = (long) range.getEnd().getValue();
         return new LongTokenRange(start, end);
     }
+
+    private static class KeyspaceReplication
+    {
+        private final ImmutableMap<LongTokenRange, ImmutableSet<Host>> myReplication;
+
+        public KeyspaceReplication(ImmutableMap<LongTokenRange, ImmutableSet<Host>> replication)
+        {
+            myReplication = replication;
+        }
+
+        public ImmutableMap<LongTokenRange, ImmutableSet<Host>> getReplication()
+        {
+            return myReplication;
+        }
+
+        @Override
+        public boolean equals(Object o)
+        {
+            if (this == o) return true;
+            if (o == null || getClass() != o.getClass()) return false;
+
+            KeyspaceReplication that = (KeyspaceReplication) o;
+
+            return myReplication != null ? myReplication.equals(that.myReplication) : that.myReplication == null;
```

Comment:
Consider to replace with `return Objects.equals(myReplication, that.myReplication);`

Reply:
Removed the class